modules/adding-a-custom-logo.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="adding-a-custom-logo-{context}"]
+= Adding a custom logo
+
+.Prerequisites
+
+* Create a 68px x 200px PNG file of the logo that you want to use.
```

Comment:
This may be overly prescriptive.  The logo can be a GIF, JPG, PNG, or SVG and is constrained to a max-height of 60px.

Reply:
@rhamilto Thank you!